### ISSUE TITLE
[WASM] Some supported ops in wasm backend are not available

### ISSUE THREAD

**Issue by akineeic** [tensorflow/tfjs#2649]
Hello,
I have encountered some issues while using the tfjs-backend-wasm package, some infos are as follows:

#### TensorFlow.js version

- @tensorflow/tfjs-core@1.4.0

- @tensorflow/tfjs-backend-wasm@1.5.1-alpha5

#### Browser version

- Google Chrome: 79.0.3945.79 (Official Build) (64-bit)

#### Describe the problem or feature request
Since 'Softmax' is not supported in wasm backend, I want to simulate a sample softmax function with ops 'Exp', 'Sum' and 'Div'. But when I try it with the [guidance](https://github.com/tensorflow/tfjs/tree/master/tfjs-backend-wasm/starter/webpack) of webpack and test in chrome's inspect, the following error occured.
![tfjs-wasm-issue](https://user-images.githubusercontent.com/44738552/71868979-6bda5380-314b-11ea-9698-48a1b44adb0d.png)

#### Code to reproduce the bug / link to feature request
I used the code in the folder [tfjs/tfjs-backend-wasm/starter/webpack](https://github.com/tensorflow/tfjs/tree/master/tfjs-backend-wasm/starter/webpack), and I upgrade the package '@tensorflow/tfjs-backend-wasm' to version:1.5.1-alpha5 since I can't set the backend to 'wasm' while using the version in the ['yarn.lock'](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/starter/webpack/yarn.lock). 

To reproduce the error, just change the [code](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/starter/webpack/src/index.js#L24) to the usage of other kernel like 'exp' or 'sum' like the followings:
`const x = tf.tensor1d([1, 2, 3]);`
`x.sum().print();  // or tf.sum(x)`

Then use commend `yarn build & yarn watch` to run the code in browser and the error occured. It seems like caused by the build or registry process or maybe I make some mistake in the usage. So I'm wondering are there any strategy  to fix the issue?

Thanks.

**Comment by akineeic** [comment]
I have checked some other ops and found more ops that are also unavailable, like 'Greater', 'Less', 'Maximum', 'Minimum', etc. But those ops appear in the file 'all_kernels.ts' which is located in the package folder 'node_module'. Then I compared with the [history](https://github.com/tensorflow/tfjs/commits/master/tfjs-backend-wasm/src/kernels/all_kernels.ts) of 'all_kernels.ts', it seems that the ops added after 'exp' is all unavailable. So I guess this issue occured maybe because the wasm related files in the folder 'wasm-out' didn't rebuild and update before the package published.

**Comment by dsmilkov** [comment]
Hi, I just updated our starter projects to depend on latest wasm AND latest core in https://github.com/tensorflow/tfjs/pull/2706. The problem you were seeing is b/c tfjs-backend-wasm was at 1.5.1 but tfjs-core was at 1.4.0 (where 'exp' was not yet modularized) and that introduced a skew.
